tests/main_test.go

```diff
@@ -91,4 +100,15 @@ type Config struct {
 	SecretUpdate bool
 	// ScaleToZero enables/disables the scale from zero test
 	ScaleToZero bool
+
+	// Namespaces to verfiy OpenFaaS provider
+	Namespaces []string
+}
+
+func FromEnv(config *Config) {
+	// read CERTIFIER_NAMESPACES variable, parse as csv string
+	namespaces, present := os.LookupEnv("CERTIFIER_NAMESPACES")
+	if present {
+		config.Namespaces = strings.Split(namespaces, ",")
```

Comment:
I think the only improvement we could make here is to trim the space from each namespace found

https://golang.org/pkg/strings/#TrimSpace

Reply:
Thanks, made the changes

Reply:
Know what, let's merge this and we can handle this as we go

Reply:
lol, github was showing me a stale diff earlier, just saw the change is there :muscle: